pom.xml

```diff
@@ -390,19 +390,26 @@ under the License.
 			<dependency>
 				<groupId>com.fasterxml.jackson.core</groupId>
 				<artifactId>jackson-annotations</artifactId>
-				<version>2.10.1</version>
+				<version>${jackson.version}</version>
 			</dependency>
 
 			<dependency>
 				<groupId>com.fasterxml.jackson.core</groupId>
 				<artifactId>jackson-databind</artifactId>
-				<version>2.10.1</version>
+				<version>${jackson.version}</version>
 			</dependency>
 
 			<dependency>
 				<groupId>com.fasterxml.jackson.core</groupId>
 				<artifactId>jackson-core</artifactId>
-				<version>2.10.1</version>
+				<version>${jackson.version}</version>
+			</dependency>
+
+			<!-- https://issues.apache.org/jira/browse/FLINK-15868 -->
+			<dependency>
+				<groupId>com.fasterxml.jackson.dataformat</groupId>
+				<artifactId>jackson-dataformat-cbor</artifactId>
```

Comment:
I would only put this into the kinesis connector pom. This is the only module that should be affected by virtue of promoting transitive dependencies via the shade-plugin.
This promotion should likely be removed in the future because it has rather subtle side-effects.

Reply:
Looks like there is consensus to keep it centralized; I prefer that as well. This dependency isn't specific to Kinesis, it comes from `aws-java-sdk-core`.

Reply:
> Looks like there is consensus to keep it centralized

Not quite; this came up during the offline discussion and we continued with the existing approach for the time-being for simplicity. Long-term we're likely to throw a jackson bom into every module that needs it, and ban older versions.
This way it is a lot easier to tell which modules are affected by the, if a new jackson dependency is added 1 of three cases happen:
a) dependency convergence fails (happens if it is not shaded)
b) enforcer check fails (old insecure versions used that we don't want) 
c) a safe version is used and shaded (hence no conflicts), in which case there is no reason to mandate another version. This also allows us to use the version that the transitive dependency was built against, which is one less uncertainty on our side.